flojoy/job_result_utils.py

```diff
@@ -17,11 +17,22 @@ def is_flow_controled(result: dict[str, Any] | DataContainer):
     return False
 
 
-def get_next_directions(result: dict[str, Any] | None) -> Optional[list[str]]:
+def get_next_directions(result: dict[str, Any] | None) -> list[str]:
     if result is None:
-        return None
-
-    return result.get(FLOJOY_INSTRUCTION.FLOW_TO_DIRECTIONS)
+        return ["default"]
+    direction = ["default"]
```

Comment:
This should return `None`  so that we can just flow outward in all of the out edges (see line 162 in studio's `captain/models/topology.py`)